mineral-ui/mineral-ui
# [Website] component docs content

closes #346 

### Description
The component pages needed context to explain intended usage. This PR moves the needle a bit towards having a documented design system.

_Note: I was going to put mineral content from wikipedia in here, and while we might do that eventually, it should probably be a separate effort. The CC license allows us to use images and copy for the most part, but we have some hoops to jump through, and we shouldn't do it incorrectly._

### Motivation and context
We had good docs on _how_ to use the components, but we needed a bit more of when and how to use the components and what their intended uses are. This PR also removes the Fill Murray content, since technically that needs attribution too now that it's public-facing.

### How to test
1. `cd mineral-ui && npm start`
2. All the components pages have been updated. Make sure I didn't mispel anything.
3. Sections to check are Intro, all the examples, When/How, and Dos and Don'ts

### Types of changes
- Docs

### Checklist




### How does this PR make you feel?
![Alt text here](https://media.giphy.com/media/14w9WNvpFKXoZO/giphy.gif)
